README.md

```diff
@@ -1,6 +1,8 @@
 # picolibrary
 ![CI](https://github.com/apcountryman/picolibrary/workflows/CI/badge.svg)
 
```

Comment:
If this blank line is removed, both badges should be displayed on the same line.